app/frontend/src/components/showHiddenContent/showHiddenContent.js

```diff
@@ -0,0 +1,42 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+  static targets = ['content'];
+
+  connect() {
+    const [errors] = this.contentTarget.getElementsByClassName('govuk-error-summary');
+    const actionEl = document.querySelector('[data-action="click->show-hidden-content#show"');
+
+    if (!errors) {
+      this.contentTarget.style.display = 'none';
+
+      ariaHidden(this.contentTarget);
+      ariaExpanded(actionEl);
+    } else {
+      actionEl.style.display = 'none';
+
+      ariaExpanded(this.contentTarget);
+      ariaHidden(actionEl);
+    }
+  }
+
+  show(event) {
+    event.preventDefault();
+
+    event.target.style.display = 'none';
+    ariaHidden(event.target);
+
+    this.contentTarget.style.display = 'block';
+    ariaExpanded(this.contentTarget);
+  }
+}
+
+const ariaHidden = (el) => {
```

Comment:
nice if these were static methods in controller

Reply:
doesnt matter for this Pr but these kind of functions are already and can be resued for other stuff, might be a nice little exercise to do another time when the todo column is sparse!